tests/integrations/tests/integrations/test_openai.py

```
@@ -1054,3 +1095,470 @@ def test_30_embedding_usage_tracking(self, openai_client, test_config):
+                args = json.loads(function_call_message.arguments)
+            else:
+                args = function_call_message.arguments
+
+            assert "location" in args, "Function call should have location argument"
+            location_lower = str(args["location"]).lower()
+            assert "boston" in location_lower, (
+                f"Location should mention Boston, got {args['location']}"
+            )
+
+    @pytest.mark.parametrize("provider,model", get_cross_provider_params_for_scenario("responses_api"))
+    def test_36_responses_streaming(self, openai_client, test_config, provider, model):
```

Ruff (0.14.2)

Unused method argument: `test_config`

(ARG002)